@shared/api/helpers/getDomainFromIssuer.ts

```diff
@@ -19,6 +19,7 @@ export const getDomainFromIssuer = async ({
   try {
     /* First, check our localStorage cache in Background to see if we've found this url before */
     ({ assetDomain } = await sendMessageToBackground({
+      activePublicKey: null,
```

Comment:
there are some instances where a user may not yet have a public key (for ex: during onboarding or before the `loadAccount` request resolves). We don't want to block these types of requests but we do want to make a conscious choice to omit the `activePublicKey` here. In these cases, let's use `null`

Reply:
are we omitting it here because the `activePublicKey` shouldn't matter for fetching a asset domain?

Reply:
Yup, exactly. Fetching an asset domain isn't anything specific to a public key